CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## Version 4.0.0
 
+* Step simpleclient to 0.16.0 from 0.10.0 - Issue #416
```

Comment:
Please follow the same format as other similar changes.

Step prometheus simpleclient from 0.10.0 to 0.16.0 - Issue #416